harness/determined/cli/job.py

```diff
@@ -1,56 +1,64 @@
 import json
 from argparse import Namespace
+from datetime import datetime
 from typing import Any, List
 
 import yaml
 
 from determined.cli import render
 from determined.cli.session import setup_session
+from determined.common import api
 from determined.common.api import authentication
 from determined.common.api.b import get_GetJobs
 from determined.common.declarative_argparse import Arg, Cmd, Group
 
 
 @authentication.required
 def ls(args: Namespace) -> None:
+    priority = True
+    config = api.get(args.master, "config").json()
+    try:
+        if config["resource_manager"]["scheduler"]["type"] == "fair_share":
```

Comment:
what if the resource pool config overrides it?

Reply:
ah good point. Will fix